lib/pa_ess/updater.ex

```diff
@@ -32,12 +43,81 @@ defmodule PaEss.Updater do
           config_engine: config_engine
         },
         audios,
+        tts_audios,
         extra_logs
       ) do
     if config_engine.scu_migrated?(scu_id) do
-      Logger.error("Error sending to new SCU, not implemented")
+      Task.Supervisor.start_child(PaEss.TaskSupervisor, fn ->
+        files =
+          Enum.map(tts_audios, fn {text, _} ->
+            Task.async(fn -> fetch_tts(text) end)
+          end)
+          |> Task.await_many()
+
+        Enum.zip([files, tts_audios, extra_logs])
+        |> Enum.each(fn {file, {text, pages}, logs} ->
+          PaEss.ScuQueue.enqueue_message(
+            scu_id,
+            {:message, scu_id,
+             %{
+               visual_zones: audio_zones,
+               visual_data: format_pages(pages),
+               audio_zones: audio_zones,
+               audio_data: [Base.encode64(file)],
+               expiration: 30,
+               tag: nil
+             }, [sign_id: id, audio: inspect(text), visual: inspect(pages)] ++ logs}
+          )
+        end)
+      end)
     else
       MessageQueue.send_audio({pa_ess_loc, audio_zones}, audios, 5, 60, id, extra_logs)
     end
   end
+
+  defp zip_pages(top, bottom) do
+    max_length =
+      Enum.map([top, bottom], fn
+        str when is_binary(str) -> 1
+        list -> length(list)
+      end)
+      |> Enum.max()
+
+    Enum.map(0..(max_length - 1), fn i ->
+      [{top, top_duration}, {bottom, bottom_duration}] =
+        Enum.map([top, bottom], fn
+          str when is_binary(str) -> {str, 6}
+          list -> Enum.at(list, i, List.last(list))
+        end)
+
+      ^top_duration = bottom_duration
```

Comment:
Yeah, I put that in there as an assertion to make sure they match. Looking at the code, I think everything is the same now (6 seconds), and the intention is to enforce that going forward. If this seems to harsh, though, we could log an error instead.

Reply:
Yeah I think we loosely standardized on 6 seconds a while back so I guess this assertion would rarely, if ever, fail. But at least in concept, it does seem a bit harsh since every message gets funneled through this process which I think means a few consecutive crashes could be pretty disruptive.